src/service/setupService.ts

```diff
@@ -171,7 +171,20 @@ export class SetupService {
             : GLOBAL_ENV_VARS.PYTHON;
         // try to get name from interpreter
         try {
-            originalPythonExecutablePath = getConfig(CONFIG.PYTHON_PATH);
+            const extension = vscode.extensions.getExtension("ms-python.python");
+            const usingNewInterpreterStorage = extension.packageJSON?.featureFlags?.usingNewInterpreterStorage;
+            if (usingNewInterpreterStorage) {

```

Comment:
Add a comment explaining that the python extension is deprecating the old path? As we are still falling back to it if the new interpreter storage is not set